.github/workflows/CD.yml

```
@@ -0,0 +1,40 @@
+name: CD
+
+on:
+    push:
+      tags:
+        - '*'
+
+env:
+  PY_VERSION: "3.11"
+
+jobs:
+
+  conda:
+    name: Build with conda and upload
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+        with:
+            fetch-tags: true
+            fetch-depth: 0 
+
+      - name: Setup conda environment
+        uses: conda-incubator/setup-miniconda@11b562958363ec5770fef326fe8ef0366f8cbf8a
+        with:
+          miniconda-version: "latest"
+          python-version: ${{ env.PY_VERSION }}
+          environment-file: conda/env_build.yml
```

Shouldn't this be:
```suggestion
          environment-file: .conda/env_build.yml
```
with the preceding `.` for the `.conda` folder?